packages/vcs/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "@tbd54566975/vcs",
+  "version": "0.1.4",
+  "description": "Verifiable Credentials",
+  "type": "module",
+  "main": "./dist/cjs/main.cjs",
+  "module": "./dist/esm/main.mjs",
+  "types": "./dist/types/main.d.ts",
+  "scripts": {
+    "build": "rimraf dist && node build/bundles.js && echo '{\"type\": \"commonjs\"}' > ./dist/cjs/package.json && tsc",
+    "lint": "eslint . --ext .js --max-warnings 0",
+    "lint:fix": "eslint . --ext .js --fix",
+    "test:node": "rimraf __tests__ && tsc -p tsconfig.test.json && mocha \"__tests__/**/*.spec.js\" --exit",
+    "test:browser": "karma start karma.conf.cjs"
+  },
+  "homepage": "https://github.com/TBD54566975/web5-js/tree/main/packages/web5-agent#readme",
```

Comment:
should point here? https://github.com/TBD54566975/web5-js/tree/main/packages/vcs

Reply:
good catch!

Reply:
done